src/ServiceWorkerCacher.ts

```diff
@@ -1,15 +1,18 @@
 import Cacher from "./Cacher";
+import Store from "./Store";
 import Manifest from "./Manifest";
 
 /** Class that caches URLs using ServiceWorker's Cache API. */
 export default class ServiceWorkerCacher implements Cacher {
     private serviceWorkerPath: string;
+    private store: Store;
     private supported: boolean;
 
     /** Create a ServiceWorkerCacher. */
     /** @param serviceWorkerPath Location of the service worker js file. */
-    public constructor(serviceWorkerPath: string = "sw.js") {
+    public constructor(store: Store, serviceWorkerPath: string = "sw.js") {
```

Comment:
This default works if you put the service worker at the root of your exploded epub, which I think is the simplest setup. It's hard to say what would be most common.

It works for the example in this repo because I have a symlink to sw.js there.

It actually doesn't work for our real use case (https://github.com/NYPL-Simplified/web-reader-landing-page/blob/master/book_index.html.template#L108), which is why I had to make it configurable.